pallets/price-tools/src/migrations/v2.rs

```diff
@@ -0,0 +1,50 @@
+// This file is part of the SORA network and Polkaswap app.
+
+// Copyright (c) 2020, 2021, Polka Biome Ltd. All rights reserved.
+// SPDX-License-Identifier: BSD-4-Clause
+
+// Redistribution and use in source and binary forms, with or without modification,
+// are permitted provided that the following conditions are met:
+
+// Redistributions of source code must retain the above copyright notice, this list
+// of conditions and the following disclaimer.
+// Redistributions in binary form must reproduce the above copyright notice, this
+// list of conditions and the following disclaimer in the documentation and/or other
+// materials provided with the distribution.
+//
+// All advertising materials mentioning features or use of this software must display
+// the following acknowledgement: This product includes software developed by Polka Biome
+// Ltd., SORA, and Polkaswap.
+//
+// Neither the name of the Polka Biome Ltd. nor the names of its contributors may be used
+// to endorse or promote products derived from this software without specific prior written permission.
+
+// THIS SOFTWARE IS PROVIDED BY Polka Biome Ltd. AS IS AND ANY EXPRESS OR IMPLIED WARRANTIES,
+// INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+// A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL Polka Biome Ltd. BE LIABLE FOR ANY
+// DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+// BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS;
+// OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+// STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE
+// USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+use frame_support::dispatch::GetStorageVersion;
+use frame_support::traits::StorageVersion;
+
+use crate::PriceInfos;
+use crate::{AggregatedPriceInfo, Pallet};
+use crate::{Config, PriceInfo};
+
+pub fn migrate<T: Config>() {
+    if Pallet::<T>::on_chain_storage_version() < StorageVersion::new(2) {
+        PriceInfos::<T>::translate::<PriceInfo, _>(
+            |_, old_price_info| -> Option<AggregatedPriceInfo> {
+                Some(AggregatedPriceInfo {
+                    buy: old_price_info,
+                    sell: PriceInfo::default(),
+                })
+            },
+        );
+        StorageVersion::new(2).put::<Pallet<T>>()
+    }
```

Comment:
Storage version isn't changed: 2 -> 2
Is it ok?

Reply:
It checks that version is lower than 2 and then set it to 2, so it's ok